ex1.py

```diff
@@ -0,0 +1,9 @@
+#area of a circle
+import math
+def Area(r):
+    return((math.pi*r**2))
```

Comment:
both parentheses are not needed in this case, to read about order of operations in python:
https://runestone.academy/runestone/books/published/thinkcspy/SimplePythonData/OrderofOperations.html

in general, you can return a value with no parentheses like: "return 5" -> this is fine
also mind the second parentheses, they have no use, the cover the first parentheses. why did you add them?